docs/pages/server-access/troubleshooting-server.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: Troubleshooting Server Access
+description: Describes common issues and solutions for access to servers.
+---
+
+This section describes common issues that you might encounter in managing access to servers
+with Teleport and how to work around or resolve them.
+
+## Starting SSH sessions fails
+
+By default, when Teleport connects to a server, the service runs as the user login that starts 
+the SSH session. If starting the session fails, it might be because the file system permissions
+for the Teleport binary don't allow other user logins to connect to the remote server.
+This issue might be caused by systems that prevent newly installed software from being executable 
+by any user.  
```

Comment:
```suggestion
When you start a new SSH session, Teleport forks itself and the child process
runs as the OS user who is attempting to connect. If the filesystem permissions
on the `teleport` binary don't include execute permissions for this user, the
session will fail to start.

This issue could be caused by systems that prevent newly installed software from
being executable by any user.  
```

Reasoning for the suggestion:
- 'by default' is unnecessary and implies that you can change this behavior. you can't. it's just how teleport works.
- incorrect file permissions do not deny access to connect, they deny access to execute